src/core/installAddon.js

```diff
@@ -36,14 +36,20 @@ import {
   UNKNOWN,
 } from 'core/constants';
 import * as addonManager from 'core/addonManager';
+import { DISCO_DATA_EXTENSION, DISCO_DATA_THEME } from 'disco/constants';
+
 
 export function installTheme(
   node, addon, { _themeAction = themeAction, _tracking = tracking } = {},
 ) {
   const { name, status, type } = addon;
   if (type === THEME_TYPE && [DISABLED, UNINSTALLED, UNKNOWN].includes(status)) {
     _themeAction(node, THEME_INSTALL);
-    _tracking.sendEvent({ action: 'theme', category: INSTALL_CATEGORY, label: name });
+    _tracking.sendEvent({
```

Comment:
I really wish the `tracking` module was smarter and these calls could be more like:

```js
if (type === THEME_TYPE && /* ... */) {
  _tracking.themeInstalled(name);
} /* ... */
```

But that isn't the case. Anyway I find the name `DISCO_DATA_THEME` a bit weird and this code is run by mobile pages as well so maybe `TRACKING_TYPE_THEME` or similar would be better?